apps/renterd/contexts/transactions/index.tsx

```diff
@@ -66,52 +55,91 @@ function useTransactionsMain() {
     },
   })
 
-  const { openDialog } = useDialog()
-
-  const siascanUrl = useSiascanUrl()
+  const { filters, setFilter, removeFilter, removeLastFilter, resetFilters } =
+    useServerFilters()
```

Comment:
return values aren't used

Reply:
Removed, as we do not support any filtering yet.